Sources/TuistCore/Models/DependencyRequirement.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+enum DependencyRequirement {
```

Comment:
What about extracting [this one](https://github.com/tuist/tuist/blob/main/Sources/TuistCore/Models/Package.swift#L9) from `Package` to the root, and reusing it from `CarthageDependency`? That'd prevent having 2 models that represent the same thing :)